rpc_jobs/rpc_openstack.yml

```diff
@@ -453,13 +453,27 @@
     image:
       - xenial:
           IMAGE: "Ubuntu 16.04.2 LTS prepared for RPC deployment"
-      - xenial_loose_artifacts:
-          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
-          # We have an MNAIO equivalent of this PM job, but we need to keep
-          # this for the push triggering for snapshot building.
-          CRON: "@monthly"
       - trusty:
           IMAGE: "Ubuntu 14.04.5 LTS prepared for RPC deployment"
+    scenario:
+      - "swift"
+    action:
+      - deploy:
+          FLAVOR: "7"
+    jobs:
+      - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
+
+# We create a separate snapshot project since we don't need all newton-aio
+# jobs triggering snapshot builds.
+- project:
```

Comment:
The `CRON: "@monthly"` parameter has been dropped, why was that? It looks like it was there to reduce the frequency of the periodic jobs created for use by the push jobs.

Reply:
Re-added -- it was removed on my first edit but only partially restored after my 2nd edit. Thanks for catching!